tests/suites/test_suite_pkcs7.data

```diff
@@ -0,0 +1,39 @@
+PKCS7 Signed Data verification #1
+depends_on:MBEDTLS_PKCS7_USE_C
+pkcs7_verify:"data_files/pkcs7.data.signed":"data_files/cli-rsa-sha256.crt":"data_files/pkcs7.data"
```

Comment:
Please explain how the test data was generated. Ideally in the form of commands in `tests/data_files/Makefile`. It's ok if the commands don't generate an identical file, for example if they're randomized of if they depend on the current time. What we want is to be able to create variations on those files, for example if a bug is discovered in a corner case later.